CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ Multitenant AWS Dynamo supports the [AWS Dynamo Java API](http://docs.aws.amazon
   
 You can write your application code against the Amazon DynamoDB interface as you would for any other application.  The implementation will manage storage of data by tenant.
 
+## 0.9.34 (April 9, 2019)
+
+* Fixed bug [querying secondary indexes may yield expected results if two tables in the same tenant have the same index name](https://github.com/salesforce/mt-dynamo/pull/342)
```

Comment:
"expected" or "unexpected"? (here and in 342)

Reply:
fixed in 8931d403bf3af2c5eb015f5e26bb58b5e0c2f9e6